docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -750,3 +750,51 @@ spec:
   tags: compliance, hardening, built-in
   platform: darwin
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Operating system up to date (macOS)
+  query: SELECT 1 FROM os_version WHERE version >= '12.5.1';
+  description: "Checks that the operating system is up to date."
+  resolution: "From the Apple menu () in the corner of your screen choose System Preferences. Then select Software Update and select Upgrade Now. You might be asked to restart or enter your password."
+  platforms: macOS
+  tags: compliance, CIS, template, CIS1.1
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic updates enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticCheckEnabled' AND value=1 LIMIT 1;
+  description: "Checks that a mobile device management (MDM) solution configures the operating system to automatically checks for updates."
+  resolution: "Go to the Software Update section of System Preferences and install the latest updates."
```

```suggestion
  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic updates."
```

Is my suggestion the correct resolution? My suggestion uses the same language as the "Automatic update downloads enabled" policy.